group.go

```diff
@@ -0,0 +1,101 @@
+package shortcut
```

Comment:
normally we put public repos in [dgit-hub](https://github.com/dgit-hub)

what do we use this repo for?

Reply:
reconciling users in the saas user check.